api/src/main/java/keywhiz/api/BatchSecretRequest.java

```diff
@@ -0,0 +1,35 @@
+package keywhiz.api;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.Objects;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableSet;
+import org.hibernate.validator.constraints.NotEmpty;
+
+/**
+ * Batch Secret Request.
+ */
+public class BatchSecretRequest {
```

Comment:
I'd recommend using [AutoValue](https://github.com/google/auto/blob/master/value/userguide/index.md) here; it handles `equals()` and `hashCode()` automatically and makes creating builders easy (though this is a simple enough class we probably don't need a builder yet). [AutomationSecretResponse](https://github.com/square/keywhiz/blob/master/api/src/main/java/keywhiz/api/AutomationSecretResponse.java) and [ClientDetailResponseV2](https://github.com/square/keywhiz/blob/master/api/src/main/java/keywhiz/api/automation/v2/ClientDetailResponseV2.java) are a couple of the Keywhiz classes that use it.